src/htmx.js

```diff
@@ -273,12 +269,15 @@ return (function () {
             return responseNode;
         }
 
+        /**
+         * @param {string} resp
+         * @returns {boolean}
+         */
         function aFullPageResponse(resp) {
-            return resp.match(/<body/);
+            return resp.match(/<body>/) !== null;
```

Comment:
Are we intentionally changing the function implementation here from returning an Array to returning a Boolean?

Reply:
Yes. The result of the function is used as a boolean in a condition. This change makes its use more precise. Probably the name of the function could be adjusted too: `isFullPageResponse`

Reply:
For reference:

var partialResponse = !aFullPageResponse(resp);

Reply:
I agree, Renaming the function will clear the intent.